pybossa/jobs.py

```diff
@@ -164,7 +165,7 @@ def get_default_jobs():  # pragma: no cover
         yield dict(name=warn_old_project_owners, args=[], kwargs={},
                    timeout=timeout, queue='low')
     yield dict(name=warm_cache, args=[], kwargs={},
-               timeout=timeout, queue='super')
+               timeout=2*timeout, queue='super')
```

Comment:
Looks like warm_cache timeout needs higher value. This could be a separate timeout configured.

Reply:
That's why I added profiling for the warm_cache. I think it could close to 600 seconds as the the update_stats part(more time consuming IMO) has finished. We don't want it to run too long because the warm_cache is kicked off every 15 minutes. Ideally, it should finish with 15min, otherwise it will queue up.

Reply:
sounds good. based on profiler results, if needed, lets have a separate timeout configured.